mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -0,0 +1,445 @@
+                    mocTop[0, range(1, nVertLevels+1)] = transportZ.cumsum()
+                    for iLat in range(1, np.size(mocLatRegional)):
+                        indlat = np.logical_and(np.logical_and(
+                                     regionCellMask == 1,
+                                     latCell >= mocLatRegional[iLat-1]),
+                                     latCell < mocLatRegional[iLat])
+                        mocTop[iLat, :] = mocTop[iLat-1, :] + \
+                                          velArea[indlat, :].sum(axis=0)
+                    # convert m^3/s to Sverdrup
+                    mocRegional = mocTop * m3ps_to_Sv
+                    mocRegional26[ncount] = np.amax(mocRegional[indLat26, :])
+                    ncount = ncount + 1
+
+                # Save to file
+                print '   save Atlantic MOC at 26.5N to yearly file...'
+                ncFile = netCDF4.Dataset(outputFileTseries, mode='w')
+                # create dimension
+                ncFile.createDimension('Time', 0)
+                # create variables
+                time = ncFile.createVariable('time', 'f4', ('Time',))
+                time.description = 'days since start of simulation'
+                fldAtlantic26 = ncFile.createVariable('mocRapidArrayMax',
+                                                      'f4', ('Time',))
+                fldAtlantic26.description = 'Max MOC Atlantic streamfunction'\
+                                            ' nearest to RAPID Array latitude'\
+                                            ' (26.5N)'
+                fldAtlantic26.units = 'Sv (10^6 m^3/s)'
+                # save variables
+                time[:] = ds.Time[ncount-12:ncount].values
+                fldAtlantic26[:] = mocRegional26[ncount-12:ncount]
+                ncFile.close()
+            else:
+                ncount = ncount + 12
+                # Read from file
+                print '     Read previously computed Atlantic MOC at 26.5N'\
+                      ' from file...'
+                ncFile = netCDF4.Dataset(outputFileTseries, mode='r')
+                mocRegional26[ncount-12:ncount] = ncFile.variables['mocRapidArrayMax'][:]
+                ncFile.close()
+
+        # Make xarray dataset
+        dsmoc = xr.Dataset({'mocRegional26': ('Time', mocRegional26), 'Time': ds.Time})
+
+        # Plot time series
+        print '   Plot Atlantic MOC at 26.5N...'
+        xLabel = 'Time [years]'
+        yLabel = '[Sv]'
+        title = 'Max Atlantic MOC at $26.5^\circ$N\n {}'.format(mainRunName)
+        figureName = '{}/mocTimeseries_{}.png'.format(plotsDirectory, 
+                      mainRunName)
+        timeseries_analysis_plot(config, [dsmoc.mocRegional26],
+                                 movingAveragePoints, title,
+                                 xLabel, yLabel, figureName,
+                                 lineStyles=['k-'], lineWidths=[1.5],
+                                 calendar=calendar)
```

Comment:
Is it possible to do all the plotting at the end in designated sections for each particular plot?  As it is now plotting occurs throughout the script, which is somewhat confusing.